scripts/apps/contacts/components/ItemList.jsx

```diff
@@ -0,0 +1,157 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {Item} from 'apps/contacts/components';
+
+/**
+ * Contact Item list component
+ */
+export class ItemList extends React.Component {
```

Comment:
imo this component could be quite generic if you pass there `Item` component as a prop